src/KOKKOS/pair_reaxff_kokkos.h

```diff
@@ -15,19 +15,21 @@
 
 #ifdef PAIR_CLASS
 // clang-format off
-PairStyle(reax/c/kk,PairReaxCKokkos<LMPDeviceType>);
-PairStyle(reax/c/kk/device,PairReaxCKokkos<LMPDeviceType>);
-PairStyle(reax/c/kk/host,PairReaxCKokkos<LMPHostType>);
+PairStyle(reaxff/kk,PairReaxFFKokkos<LMPDeviceType>);
+PairStyle(reaxff/kk/device,PairReaxFFKokkos<LMPDeviceType>);
+PairStyle(reaxff/kk/host,PairReaxFFKokkos<LMPHostType>);
+PairStyle(reax/c/kk,PairReaxFFKokkos<LMPDeviceType>);
```

Comment:
This code seems duplicated?

Reply:
This is intentional. This way we stay backward compatible.
Input files that have 'reax/c' instead of 'reaxff' will continue to work.
Please see, e.g., src/REAXFF/pair_reaxff.h that has the same kind of duplicated line.

Reply:
Ah I missed that, thanks.